doc/api/child_process.md

```diff
@@ -1211,6 +1211,10 @@ may not actually terminate the process.
 
 See kill(2) for reference.
 
+On Windows, the child process is forcefully terminated because
+`SIGKILL` and `SIGTERM` signals are not available.
```

Comment:
@joaolucasl I was thinking something like this:

```suggestion
On Windows, where POSIX signals do not exist, the `signal` argument will be
ignored, and the process will be killed forcefully and abruptly (similar to
`'SIGKILL'`).
```

Reply:
I Iike that. Might work!